cfme/containers/project.py

```diff
@@ -100,6 +110,22 @@ def resetter(self):
             self.view.toolbar.view_selector.select("Summary View")
 
 
+@navigator.register(Project, 'Dashboard')
+class Dashboard(CFMENavigateStep):
+    VIEW = ProjectDetailsView
```

Comment:
Shouldn't this be `ProjectDashboardView`, the new view class created above?

Reply:
Yes, you are right.
Change it to ProjectDashboardView.